zh/deploy-cluster-on-arm64.md

```diff
@@ -25,6 +29,10 @@ tidbBackupManagerImage: pingcap/tidb-backup-manager-arm64:v1.3.1
 
 ## 部署 TiDB 集群
 
+> **注意：**
```

Comment:
Add two sections: 
>=5.4.2
<5.4.2
Similar for the TiDB Operator.

Reply:
Done